yktrace/src/tir.rs

```diff
@@ -314,6 +318,11 @@ struct VarRenamer {
     stack: Vec<u32>,
     /// Current offset used to rename variables.
     offset: u32,
+    /// Accumulator keeping track of total amount of variables used. Needed to use different
```

Comment:
I think this would read better as "Accumulator keeping track of the total *number* of variables used"

I might be wrong, but "amount" feels like it implies an uncountable quantity?

Reply:
Fixed in 61cadb7